### ISSUE TITLE
MathJax not working in offline mode

### ISSUE THREAD

**Issue by akukas** [benweet/stackedit#903]
When I run StackEdit without a connection to the internet, LaTeX equations in my document aren't rendered.

**Comment by benweet** [comment]
Yeah, MathJax is not cached at the moment. Will have to work on that.
